packages/mediapipe-task-text/pubspec.yaml

```diff
@@ -0,0 +1,22 @@
+name: mediapipe_text
+description: A new Flutter package project.
+version: 0.0.1
+repository: https://github.com/google/flutter-mediapipe
```

Comment:
issue_tracker and probably good to include for now @dcharkes's comment from #11 publish_to: none for now